aws/studio_hpo_container_setup.ipynb

```diff
@@ -0,0 +1,721 @@
+{
```

Comment:
<p>Which workflow and what's step 1? Probably would help to link to README</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/147/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>